README.md

```diff
@@ -1,2 +1,75 @@
 # Js_todo_app
-A JavaScript todo app built with Webpack
+
+A JavaScript todo app built with webpack while using DOM manipulations
+
+## Description
+
+In this project we continue practicing DOM manipulation by dynamically rendering a simple Todo list app. By the end, we are going to be using JavaScript to generate most of the app's content! The project has two directories namely 'src' and 'dist' other than the root directory.
+
+## src
+
+'src' is the directory where all the .js module files, .css files, resources and index.js files are stored.
+
+## dist
+
+'dist' is the directory where the webpack bundling output files (main.js and other loads) and index.html file are stored.
+
+## Webpack
+
+Webpack is used as module bundler and is installed using npm installer. A config file 'webpack.config.js' is used to define rules and plugins.
+
+* Loaders installed and included in the mebpack.config.js file are: 'style-loader', 'css-loader', and 'file-loader';
+
+## npm
+
+npm is used as the installer and package manager. terminal command 'npm init' is used to initialize npm locally and generate package.json file.
+
+## index.js
+
+The 'index.js' file in the src directory is the main file which imports all the necessary module files and adds logic (functions) to display them in required manner. All html content inside the body except the first outer div element "content", are generated by this DOM file.
+
+## Live Demo
+
+* [Live Demo Link](https://davidosky007.github.io/Js_todo_app/)
+
+## Built With
+
+* HTML
+* CSS
+* Javascript
+* Bootstrap
+* webpack
+* Npm
+* file loader
+* style loader
+* css loader
+
+### Install and Setup
+
+* Clone repo to your local `https://github.com/Davidosky007/Js_todo_app.git`
+* cd into the cloned repo
+* Open a terminal in the root of the app
```

Comment:
- These instructions are not clear. What do I do next after changing the directory? How do I open the app?
- Always leave no room for assumptions. Give explicit commands that the user can use to start the app. In my case, I cloned and opened the app in `vscode` then ran `live-server dist` to open the `index.html` file located in the `dist` folder.
- Update this instruction